playbooks/openstack-deploy_caasp.yml

```diff
@@ -178,8 +178,8 @@
         - name: Add vip with cidr in extravars
           lineinfile:
             path: "{{ socok8s_extravars }}"
-            regexp: "^suse_osh_deploy_vip_with_cidr.*"
-            line: "suse_osh_deploy_vip_with_cidr: {{ _vipip.stdout }}/24"
+            regexp: "^socok8s_deploy_vip_with_cidr.*"
+            line: "socok8s_deploy_vip_with_cidr: {{ _vipip.stdout }}/24"
```

Comment:
please see my comment on naming the vip. I am fine with this, but if we have multiple, it might be better to give it the right name directly.

Reply:
Currently I don't see that change in ucp branch so not sure we need to worry about that aspect. This can be enhanced in post-merge tasks.